packages/react/src/primitives/types/menu.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { ButtonProps } from './button';
+import { FlexProps } from './flex';
+import { Sizes } from './base';
+
+export interface MenuProps extends FlexProps {
+  /**
+   * Alignment of menu against trigger
+   * @default "start"
+   */
+  align?: 'start' | 'center' | 'end';
```

Comment:
is there a better name for this property? can be easily confused with other `*align` style properties

Reply:
Agreed, do you have any suggestions? Perhaps `menuAlign`?

Reply:
Updated to `menuAlign`

Reply:
thanks <3